src/NetworkTest/testQuality/helpers/subscriberMOS.ts

```diff
@@ -74,22 +74,27 @@ function calculateAudioScore(
     return roundTripTime;
   };
 
+  const getDelay = (): number => {
+    const roundTripTime = getRoundTripTime();
+    const delay = (roundTripTime * MS_PER_SEC) / 2;
```

Comment:
Dividing the roundTripTime by 2 based on this:
https://github.com/opentok/gru/blob/develop/src/main/java/com/tokbox/gru/extractors/AudioScoreAlgorithm.java#L57
and 
[http://www.hamilton.ie/dwmalone/assem_et_al.pdf](http://www.hamilton.ie/dwmalone/assem_et_al.pdf)

<img width="276" alt="Screen Shot 2020-09-07 at 11 28 08 AM" src="https://user-images.githubusercontent.com/10964589/92397478-40a26c80-f0fd-11ea-953c-1d1dd7f7461c.png">

Reply:
looks right to me.